packages/cardhost/app/controllers/cards/view.js

```diff
@@ -29,4 +30,15 @@ export default class ViewCardController extends Controller {
     this.themerOptions.push({name: 'Custom theme'})
     this.router.transitionTo('cards.view', this.model, { queryParams:{ editingCss: true } })
   }
+
+  @action
+  save() {
+    this.model.save()
+  }
+
+  @action
+  closeEditor() {
+    this.cssModeToggle.setEditingCss(false);
+    this.router.transitionTo('cards.view', this.model, {queryParams:{editingCss: false}})
```

Comment:
I don't know how to clear the query params altogether. Any ideas?

Reply:
I believe this works

```
this.router.transitionTo('cards.view', this.model, {queryParams: { editingCss: undefined }})
```